roles/edpm_nova_compute/defaults/main.yml

```diff
@@ -18,6 +18,7 @@
 # All variables intended for modification should be placed in this file.
 
 # All variables within this role should have a prefix of "edpm_nova_compute"
+edpm_service_net_map: "{{ service_net_map }}"
```

Comment:
this is more a playbook var then a role var but i understand why you are adding this here.
this is not actully part of the edpm_nova_compute role which is why it does nto have edpm_nova_compute
as the prfix so this is correct.